### PR TITLE
Allow item limits for individual cards

### PR DESCRIPTION
**Pull request by svengato**
Allow separate item limits in the News, Events, and Twitter cards. If not specified, they use existing card_item_limit.

### CODE REVIEW

**Review by alancleary — CHANGES_REQUESTED** [review]
The code changes look good. Please add a commit that updates the readme to document this functionality.

**Review by alancleary — APPROVED** [review]
Thanks for updating the readme!

**Comment by alancleary** [comment]
@svengato Do you mind updating the theme submodule in the starter site?

**Comment by svengato** [comment]
I updated it, but `git push` gives these errors. Maybe I should make a fresh clone and try again?
```
Enumerating objects: 5, done.
Counting objects: 100% (5/5), done.
Delta compression using up to 24 threads
Compressing objects: 100% (3/3), done.
Writing objects: 100% (3/3), 342 bytes | 342.00 KiB/s, done.
Total 3 (delta 1), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (1/1), completed with 1 local object.
remote: error: GH006: Protected branch update failed for refs/heads/main.
remote: error: At least 1 approving review is required by reviewers with write access.
To https://github.com/legumeinfo/jekyll-starter-legumeinfo.git
 ! [remote rejected] main -> main (protected branch hook declined)
error: failed to push some refs to 'https://github.com/legumeinfo/jekyll-starter-legumeinfo.git'
```

**Comment by alancleary** [comment]
> remote: error: GH006: Protected branch update failed for refs/heads/main.
remote: error: At least 1 approving review is required by reviewers with write access.

The main branch is protected so you have to push to another branch and make a PR. And the PR can't be merged until someone approves it.

**Comment by svengato** [comment]
Pull request made.

**Pull request merged**
